src/index.html

```diff
@@ -18,5 +31,82 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <a href="logo" class="images">
+        <img
+        src="images/logo.png" alt="Moyo"
+        class="images"
+      />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="Apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="Samsung"
+              class="nav__link"
+            >
+              SAMSUNG
```

Comment:
Text transformation should be handled using CSS. Consider using CSS to transform text to uppercase instead of directly using uppercase letters in HTML.